tmt/utils.py

```diff
@@ -309,8 +309,31 @@ class StructuredFieldError(GeneralError):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 def quote(string):
-    """ Surround a string with double quotes """
-    return f'"{string}"'
+    """ Quote given string in a shell-compatible way """
+    return shlex.quote(string)
+
+
+def quoted_env(env_vars):
```

Comment:
Isn't this doing almost the same as `tmt.utils.dict_to_shell()`?

Reply:
it is gone, it was a stupd idea ;(